src/ln/channelmanager.rs

```diff
@@ -685,10 +685,14 @@ impl ChannelManager {
 						Ok(funding_msg) => {
 							(chan, funding_msg.0, funding_msg.1)
 						},
-						Err(_e) => {
-							//TODO: Push e to pendingevents
+						Err(e) => {
+							let mut pending_events = self.pending_events.lock().unwrap();
+							pending_events.push(events::Event::DisconnectPeer {
```

Comment:
This should be more like HandleError, not DisconnectPeer, and we should handle the error as any other error instead, I think.